debian/patches/0074-vt-enable-keyframe-only-decoding.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/libavcodec/videotoolbox.c b/libavcodec/videotoolbox.c
+index c234b18f59..72e8137b35 100644
+--- a/libavcodec/videotoolbox.c
++++ b/libavcodec/videotoolbox.c
+@@ -983,6 +983,14 @@ static int videotoolbox_start(AVCodecContext *avctx)
+             av_log(avctx, AV_LOG_ERROR, "VideoToolbox reported invalid data.\n");
+             return AVERROR_INVALIDDATA;
+         case 0:
++            if (avctx->skip_frame >= AVDISCARD_NONKEY) {
++                status = VTSessionSetProperty(videotoolbox->session,
++                                              kVTDecompressionPropertyKey_OnlyTheseFrames,
++                                              kVTDecompressionProperty_OnlyTheseFrames_KeyFrames);
```

Comment:
What would happen without this videotoolbox setting?

Reply:
It would still work, this is added to make sure the decoder works as intended.